WebInkEnhancement/explainer.md

```diff
@@ -95,14 +94,11 @@ class InkRenderer {
         });
 
         if (this.presenter)
-            this.presenter.setLastRenderedPoint(evt.x, evt.y);
+            this.presenter.setLastRenderedPoint(evt, rgba(0, 0, 255, 0.5), 2);

```

Comment:
The idea behind keeping PenStrokeStyle being that it would make it easier to expand the number of properties that the web dev could control later on? Might also be slightly easier for the web dev if the style is changing frequently I suppose, but that seems like it would be an uncommon case.

Reply:
Yes, easier expansion, and grouping of concepts are the motivations (i.e. we don't want to have to continue to add parameters should other interesting properties arise)